<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stuinformanage</class>
 <widget class="QWidget" name="stuinformanage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SEARCH BY SNO:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="infor_addButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>STUDENT INFOEMATION MANAGE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="infor_input_IDlineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="infor_querypushButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SEARCH</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SNO:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="infor_outputlineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>NAME:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SEX:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>AGE:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="infor_namelineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="infor_agelineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>360</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>CLASS:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>410</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>MAJOR:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>HOME ADDRESS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>260</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>PHONE NUMBER</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="infor_classlineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>410</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="infor_adresslineEdit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="infor_phonenumlineEdit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="infor_deletepushButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>DELETE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="infor_updatepushButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>140</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>UPDATE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="infor_backpushButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>80</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="infor_see_inforpushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SHOW ALL STUDENTS' INFORMATION</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>360</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OCCUPATION</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="infor_passwordEdit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>310</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>310</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>PASSWORD</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>360</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>student</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>teacher</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="major_comboBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>410</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>CST</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>IS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>IT</string>
    </property>
   </item>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>male</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>female</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
